### ISSUE TITLE
When you import a type, it does not show an error

### ISSUE THREAD

**Issue by WorkingDevJs** [import-js/eslint-import-resolver-typescript#232]
I use it in a monorepo, it works perfectly except when I import a type

### This does not mark an error even though the file does not exist
`import type { CompanyDetailCommentProps } from '~/pages/company/CompanyDetail';`

### eslintrc.json
```
{
	"extends": ["@crac/eslint-config"],
	"overrides": [
		{
			"files": ["*.ts", "*.tsx"], // Your TypeScript files extension
			"parserOptions": {
				"project": ["packages/*/tsconfig.json", "apps/*/tsconfig.json", "./tsconfig.json"]
			}
		}
	],
	"settings": {
		"react": {
			"version": "detect"
		},
		"import/parsers": {
			"@typescript-eslint/parser": [".ts", ".tsx"]
		},
		"import/resolver": {
			"typescript": {
				"project": ["packages/*/tsconfig.json", "apps/*/tsconfig.json", "./tsconfig.json"]
			}
		}
	}
}
```

- "eslint-import-resolver-typescript": "3.5.3"
- "eslint-plugin-import": "2.27.5"

**Comment by Haxos** [comment]
Hello @WorkingDevJs 

I had the exact same problem. After some digging, I found in `eslint-plugin-import` the following code in `lib/rules/no-unresolved.js`:
```js
// ignore type-only imports and exports
if (node.importKind === 'type' || node.exportKind === 'type') {
  return;
}
```
After commenting the code, it did threw the error for `import type`.

I've also found a thread about this issue https://github.com/import-js/eslint-plugin-import/issues/2500

**Comment by JounQin** [comment]
Duplicate of https://github.com/import-js/eslint-plugin-import/issues/1675
